.github/workflows/changed-files.yaml

```diff
@@ -117,12 +85,9 @@ jobs:
           done
           (echo -n "transformed-output="; jq -c -n "\$ARGS.named | to_entries | map({"key": .key, "value": .value[]}) | from_entries | $TRANSFORM_EXPR" "${file_args[@]}") | tee --append "$GITHUB_OUTPUT"
 
-      - name: Telemetry summary
-        id: telemetry-summary
+      - name: Telemetry summarize
+        uses: rapidsai/shared-actions/telemetry-dispatch-write-summary@telemetry-dispatch-actions
+        continue-on-error: true
         if: always()
-        uses: ./shared-actions/telemetry-summarize
         with:
-          traceparent: "${{ inputs.traceparent }}"
-          ca_cert: "${{secrets.OTEL_EXPORTER_OTLP_CA_CERTIFICATE}}"
-          client_cert: "${{secrets.OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE}}"
-          client_key: "${{secrets.OTEL_EXPORTER_OTLP_CLIENT_KEY}}"
+          cert_concat: "${{ secrets.OTEL_EXPORTER_OTLP_CA_CERTIFICATE }};${{ secrets.OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE }};${{ secrets.OTEL_EXPORTER_OTLP_CLIENT_KEY }}"
```

Comment:
It looks like the ending newline was lost here. I also noticed some inconsistencies here in https://github.com/rapidsai/shared-actions/pull/21. We should create a build-infra task to add a pre-commit hook and style check job to this repo and `shared-actions` to keep whitespace normalized.